Preview fails when setting FunctionApp/AppService siteConfig.use32BitWorkerProcess property

Preview fails when setting siteConfig.use32BitWorkerProcess property on a FunctionApp or AppService. It looks like the tf mapping is off by a character based on the error message: `error: azure:appservice/appService:AppService resource 'abc-stage-clbdir' has a problem: site_config.0: invalid or unknown key: use32_bit_worker_process`

use32_bit_worker_process looks like it should be passed down as [use_32_bit_worker_process](https://www.terraform.io/docs/providers/azurerm/r/app_service.html#use_32_bit_worker_process)

Adding a test for this in `pulumi-terraform` shows that it is the problem:

```
=== RUN   TestPulumiToTerraformName
--- FAIL: TestPulumiToTerraformName (0.00s)
	Error Trace:	names_test.go:32
	Error:		Not equal: "use_32_bit_worker_process" (expected)
        	        != "use32_bit_worker_process" (actual)
```

@lukehoban: is it sufficient here to override the `Name` of the property in the mapping of the resources in question to fix this?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,8 +1,8 @@
 ## 0.17.2 (Unreleased)
 
-### Improvements
-
 - Updated to v1.23.0 of the AzureRM Terraform Provider.
+
+- Fixed a bug where AppService or FunctionApp could not be created with `use32BitWorkerProcess` set to `true`.
 
 ## 0.17.1 (Released March 6, 2019)
 

--- a/resources.go
+++ b/resources.go
@@ -235,6 +235,15 @@ func Provider() tfbridge.ProviderInfo {
 					// Max length of a functionapp name is 60.
 					// This was discovered directly through the portal.
 					azureName: AutoNameWithMaxLength(azureName, 60),
+					"site_config": {
+						Elem: &tfbridge.SchemaInfo{
+							Fields: map[string]*tfbridge.SchemaInfo{
+								"use_32_bit_worker_process": {
+									Name: "use32BitWorkerProcess",
+								},
+							},
+						},
+					},
 				}},
 			// Automation
 			"azurerm_automation_account":               {Tok: azureResource(azureAutomation, "Account")},

--- a/resources.go
+++ b/resources.go
@@ -205,7 +205,20 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_application_insights_api_key": {Tok: azureResource(azureAppInsights, "ApiKey")},
 
 			// App Service
-			"azurerm_app_service":                         {Tok: azureResource(azureAppService, "AppService")},
+			"azurerm_app_service": {
+				Tok: azureResource(azureAppService, "AppService"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"site_config": {
+						Elem: &tfbridge.SchemaInfo{
+							Fields: map[string]*tfbridge.SchemaInfo{
+								"use_32_bit_worker_process": {
+									Name: "use32BitWorkerProcess",
+								},
+							},
+						},
+					},
+				},
+			},
 			"azurerm_app_service_custom_hostname_binding": {Tok: azureResource(azureAppService, "CustomHostnameBinding")},
 			"azurerm_app_service_plan": {
 				Tok: azureResource(azureAppService, "Plan"),

